tools/clone_flutter.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+set -x
+
+if [ -z $ENGINE_PATH ]
+then
+  echo "This script is aimed to be run on CI environments. Do not run locally."
+  exit 1
+fi
+
+# Go to the engine git repo to get the date of the latest commit.
+cd $ENGINE_PATH/src/flutter
+
+# Special handling of release branches.
+ENGINE_BRANCH_NAME=`git branch | grep '*' | cut -d ' ' '-f2'`
```

Comment:
Nit: no need for the quotes around `-f2`

Reply:
done. thanks for the suggestions!